ignite/pkg/dirchange/dirchange.go

```diff
@@ -60,7 +60,7 @@ func HasDirChecksumChanged(checksumCache cache.Cache[[]byte], cacheKey string, w
 // ChecksumFromPaths computes the md5 checksum from the provided paths.
 // Relative paths to the workdir are used. If workdir is empty, string paths are absolute.
 func ChecksumFromPaths(workdir string, paths ...string) ([]byte, error) {
```

Comment:
This change will invalidate all cache. We shouldn't backport this and leave it for v29, where it's fine to do it.